internal/kafka/internal/presenters/cluster.go

```diff
@@ -18,11 +18,22 @@ func PresentEnterpriseCluster(cluster api.Cluster, fleetShardParams services.Par
 		})
 	}
 
-	c := public.EnterpriseCluster{
+	c := public.EnterpriseClusterRegistrationResponse{
 		ClusterId:            cluster.ClusterID,
 		Status:               cluster.Status.String(),
 		FleetshardParameters: fsoParams,
 	}
 
 	return c, nil
 }
+
+func PresentEnterpriseCluster(cluster api.Cluster) public.EnterpriseCluster {
+	reference := PresentReference(cluster.ID, cluster)
+	return public.EnterpriseCluster{
+		Id:        cluster.ID,
+		Status:    cluster.Status.String(),
+		ClusterId: cluster.ClusterID,
+		Kind:      reference.Kind,
+		Href:      reference.Href,
+	}
+}
```

Comment:
With this, we are returning two ids:
- the internal id with cluster.ID
- and the ocm id with cluster.clusterID

I feel like we could drop cluster.ID and just return cluster.clusterID in its place since this is unique.
```suggestion
func PresentEnterpriseCluster(cluster api.Cluster) public.EnterpriseCluster {
	reference := PresentReference(cluster.ClusterID, cluster)
	return public.EnterpriseCluster{
		Id:        cluster.ClusterID,
		Status:    cluster.Status.String(),
		ClusterId: cluster.ClusterID,
		Kind:      reference.Kind,
		Href:      reference.Href,
	}
}
```

Reply:
To avoid consumers of the API asking themselves questions like: what is this ID and what is that ID and where do I use one over the other.

Reply:
why do both `Id` and `ClusterId` return `cluster.ClusterID`?

Reply:
@pb82 the reasoning for this suggestion is 
```
To avoid consumers of the API asking themselves questions like: what is this ID and what is that ID and where do I use one over the other.
```
mainly to avoid users having to think of the two IDs and just use one. And naturally a user will use the `.id` and not `.cluster_id`.

Reply:
@machi1990 - based on your suggestion - do we need to return internal id from the db in the `List` function or just `cluster_id` and `status`?

Reply:
the description states that. yeah. I removed the internal ID

Reply:
thanks